### PR TITLE
[`Docs` / `Awq`] Add docs on exllamav2 + AWQ

### PR DESCRIPTION
**Pull request by younesbelkada**
# What does this PR do?

cc @IlyasMoutawwakil @ArthurZucker

### CODE REVIEW

File: docs/source/en/quantization.md [hunk 1]
```diff
@@ -196,6 +196,45 @@ The parameter `modules_to_fuse` should include:
...
+```python
+import torch
+from transformers import AutoModelForCausalLM, AutoTokenizer, AwqConfig
+
+quantization_config = AwqConfig(version="exllama")
+
+model = AutoModelForCausalLM.from_pretrained(
+    "TheBloke/Mistral-7B-Instruct-v0.1-AWQ",
+    quantization_config=quantization_config,
+    device_map="auto" or torch.device("cuda"),
```
Comment (ArthurZucker) [inline docs/source/en/quantization.md]: ?

Comment (younesbelkada) [inline docs/source/en/quantization.md:218]:
```suggestion
    device_map="auto",
```

**Review by ArthurZucker — APPROVED** [review]

File: docs/source/en/quantization.md (hunk 1 above)
Comment (younesbelkada) [reply docs/source/en/quantization.md]: removed

**Comment by IlyasMoutawwakil** [comment]
lgtm thanks @younesbelkada

**Pull request merged**